hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.html

```diff
@@ -414,6 +415,7 @@
     <td>{underReplicatedBlocks}</td>
     <td>{decommissionOnlyReplicas}</td>
     <td>{underReplicateInOpenFiles}</td>
+    <td>{decommissionDuration}</td>
```

Comment:
Give the time unit here will be better for end user? Such as `<td>{decommissionDuration} ms</td>`